lib/lambda_ethereum_consensus/p2p/gossip/operations_collector.ex

```diff
@@ -41,35 +40,41 @@ defmodule LambdaEthereumConsensus.P2P.Gossip.OperationsCollector do
     "bls_to_execution_change"
   ]
 
-  def start_link(opts), do: GenServer.start_link(__MODULE__, opts, name: __MODULE__)
-
   def start() do
```

Comment:
Let's call this subscribe_to_topics, unless we're doing the `start` thing in all the rest. The name should describe what it does, specially if there are side-effects involved. Also, any reason to `raise` instead of returning an error?

Reply:
I thought of using raise because if it fails, we would want to restart. But maybe we can handle that in the caller module. I have created an issue for handling subscription errors so I'll return an error message here and leave that for a different PR.